Library/Homebrew/global.rb

```diff
@@ -7,16 +7,15 @@
 require "fileutils"
 require "json"
 require "json/add/exception"
-require "ostruct"
 require "forwardable"
+require "set"
```

Comment:
This need to be elevated (it was a transitive require of `i18n` and caused errors without it), and i thought here made the most sense. (It's a moot [point](https://docs.rubocop.org/rubocop/cops_lint.html#lintredundantrequirestatement) in Ruby 3.2, though I imagine that's a long way off for us.)